pkg/bgpv1/gobgp/conversions.go

```diff
@@ -96,6 +101,71 @@ func ToAgentPaths(paths []*gobgp.Path) ([]*types.Path, error) {
 	return ps, nil
 }
 
+func toGoBGPFamily(family types.Family) *gobgp.Family {
+	return &gobgp.Family{
+		Afi:  toGoBGPAfi(family.Afi),
+		Safi: toGoBGPSafi(family.Safi),
+	}
+}
+
+func toGoBGPAfi(a types.Afi) gobgp.Family_Afi {
```

Comment:
Inverse functions to these (`toAgentAfi`, `toAgentSafi`) are located in `gobgp/state.go` ATM, maybe this would be a good opportunity to consolidate and move them into `conversions.go` as well?